core/domain/exp_services.py

```diff
@@ -489,6 +489,17 @@ def apply_change_list(exploration_id, change_list):
             elif (
                     change.cmd ==
                     exp_domain.CMD_MIGRATE_STATES_SCHEMA_TO_LATEST_VERSION):
+                # Check that the version that the user is trying to migrate
+                # to is the latest version.
+                target_version_is_current_state_schema_version = (
+                    change.to_version ==
+                    python_utils.UNICODE(feconf.CURRENT_STATE_SCHEMA_VERSION))
+                if not target_version_is_current_state_schema_version:
+                    raise Exception(
+                        'Expected to migrate to the latest state schema '
+                        'version %s, received %s' % (
+                            feconf.CURRENT_STATE_SCHEMA_VERSION,
+                            change.to_version))
```

Comment:
@kaylahardie, the `continue` statement below is superfluous because there is nothing to execute after the `continue`. I think you can remove it.
This will fix the backend coverage as well.

Reply:
Thanks so much!